blogs/docker-ci-tricks-2/blog.md

```diff
@@ -179,14 +215,21 @@ WORKDIR /home/ci-user/git-machete
 
 ```
 
-The trick is to create a new user with the same user and group ID as your user on the host.
+In order to be able to build docker for local runs as non-root user and for the CI/CD runs on CircleCi's host machines as root user,
```

Comment:
Use uppercase `Docker` for the name of the software (like here),
and lowercase `docker` for the executable (mostly in code snippets)